src/js/components/jtable/TableCell.jsx

```diff
@@ -24,34 +37,40 @@ export class TableCell extends Component {
             classNames.push(className);
         }
 
+        const {
+            linkProps,
+            tagOrComponent
+        } = generateLink('div', href, linkTo);
```

Comment:
I guess, I find it really odd that we're using a `TableCell` as a random "do-everything" container, that's not an approach that I like at all. I wrote a bunch of other stuff, but it was irrelevant, so I'll leave it at that.